Updated the Synology instructions (#867)
jellyfin/jellyfin.org: docs/general/installation/synology.md

Click `Next` to proceed to the next step.
![Creating the Container](/images/docs/install-synology-2.png)

#### Port Settings

Allocate ports to the container. Jellyfin uses port 8096 for the web interface and 1900 for DLNA by default. It is recommended to use the default ports if they are not being used by other services.

#### Volume Settings

This setting maps directories on the host to within the container. Use this setting to allow Jellyfin access to media and a place to store application data.
To add a volume, click "Add Folder" and select the folder desired. The mount point is set in the middle column and the directory will be accessible at this path within the container. For media files, `/media` can be used, and for config files, `/config` can be used.

#### Network Settings

Set how the container connects to the network. DLNA requires `Host` mode to function.

#### Settings for advanced users

- Enviroment Variables

#### Example

Your settings should look like this:
![Advanced Settings](/images/docs/install-synology-3.png)
![Advanced Settings](/images/docs/install-synology-4.png)
Click `Next` to proceed to the next step.

> Settings can be reviewed at this screen. Check the `Run this container after the wizard is finished` checkbox and click `Apply` if everything looks correct. The container should now be shown in the `Container` tab.

Reviewer: ```suggestion
Settings can be reviewed on this screen. Check the `Run this container after the wizard is finished` checkbox and click `Apply` if everything looks correct. The container should now be shown in the `Container` tab.
```

sidebar_position: 4
---

# Install on Synology

> 

Reviewer 2: I navigate towards including a disclaimer pointing towards our hardware selection guide with a clear warning that Prebuild nas systems are generally discouraged to be used.

```suggestion
:::caution Pre-built NAS Devices

Many pre-built NAS devices are underpowered. We generally do not recommend running Jellyfin on those devices.
See: [Hardware Selection](https://jellyfin.org/docs/general/administration/hardware-selection#normal-server) for more information.

:::
```

sidebar_position: 4
---

> # Install on Synology

Reviewer 3: ```suggestion
# Installation on Synology
```

:::caution Pre-built NAS Devices

Many pre-built NAS devices are underpowered. We generally do not recommend running Jellyfin on those devices.
See: [Hardware Selection](https://jellyfin.org/docs/general/administration/hardware-selection#normal-server) for more information.

:::
For [Synology](https://www.synology.com/en-us/dsm), Jellyfin is installed using Docker. For this guide, we will use the Synology Container Manager to install Jellyfin.

## Prerequisites

Everything is done through the Synology web interface.
This guide assumes you have already set up your Synology NAS and have access to the web interface. And your Synology NAS is running DSM 7.0 or newer.

> If you are running an older version of DSM, you can use the guide at the bottom.

Author: The Old instructions only apply for machines befor 2016 so i scraped them.

Install the "Container Manager" package from the Synology Package Center.
To do this you need to open the Package Center and search for "Container Manager" and install it.

## Installation

The installation is done with the Synology Container Manager.
If after the installation of the "Container Manager" you don´t see the icon in the main menu, you can find it in when clicking on the top left corner of the main menu.

### Downloading the Jellyfin Image

Navigate to the "Registry" tab and search for "Jellyfin". You should see the official jellyfin/jellyfin image. Click on it and then click "Download".

![Downloading the Image](/images/docs/install-synology-10.png)

> A new window will open, there you can select a version of Jellyfin. We recommend using the latest version. Click "Apply".

Reviewer 3: ```suggestion
A new window will open and a Jellyfin version can be selected for install. The latest version is recommended. Click `Apply` after selection a version.
```

![Downloading the Image](/images/docs/install-synology-11.png)

> After a few seconds, the image will be downloaded and you can find it in the "Image" tab.

Reviewer 3: ```suggestion
After the image is downloaded, it can be found in the `Image` tab.
```

### Creating the Container

> Navigate to the "Container" tab and click "Create".

Reviewer 3: ```suggestion
Navigate to the `Container` tab and click `Create`.
```

> Select the "Jellyfin" image and give the container a name. We recommend using "Jellyfin" as the name. You can also set the container to auto-start if you want Jellyfin to start when the NAS boots.

Reviewer 3: ```suggestion
Select the `Jellyfin` image and give the container a name. This is mainly for identification purposes and can be set to anything desired. `Auto-start` can be enabled to automatically start Jellyfin when the NAS boots.
```
I don't see any "auto-start" options in the included images. Please clarify

> Also, you can set the resource limits for the container. We recommend that you let the container use all available resources.

Reviewer 3: ```suggestion
Resource limits can also be set for the container. It is recommended that all CPU resources and at least 4GB of ram be allocated to the Jellyfin container.
```

> Click "Next".

Reviewer 3: ```suggestion
Click `Next` to proceed to the next step.
```

![Creating the Container](/images/docs/install-synology-12.png)

> **Port Settings**

Reviewer 3: ```suggestion
#### Port Settings

```

> You need to set the port settings for the container. Jellyfin uses port 8096 for the web interface and 1900 for DLNA. You can set the container to use the same ports as the host or use different ports. We recommend using the same ports as the host.

Reviewer 3: ```suggestion
Allocate ports to the container. Jellyfin uses port 8096 for the web interface and 1900 for DLNA by default. It is recommended to use the default ports if they are not being used by other services.
```

> **Volume Settings**

Reviewer 3: ```suggestion
#### Volume Settings

```

> You need to set the volume settings for the container. Jellyfin needs access to your media files and configuration files.

Reviewer 3: ```suggestion
This setting maps directories on the host to within the container. Use this setting to allow Jellyfin access to media and a place to store application data.
```
A recommendation on how a typical setup should look like should be included.

> To add a volume, click "Add Folder" and select the folder you want to use. In the middle column, you need to set the mount path. For the media files, you can use "/media" and for the configuration files, you can use "/config".

Reviewer 3: ```suggestion
To add a volume, click "Add Folder" and select the folder desired. The mount point is set in the middle column and the directory will be accessible at this path within the container. For media files, `/media` can be used, and for config files, `/config` can be used.
```
There should also be recommendations on where the config directory should be stored on the host system.

> **Network Settings**

Reviewer 3: ```suggestion
#### Network Settings
```

> You need to set the network settings for the container. Jellyfin needs access to the network to stream media and use DLNA. You can use "Bridge" mode if you are running multiple instances of Jellyfin or "Host" mode if you are only running one instance. We recommend using "Host" mode.

Reviewer 3: ```suggestion
Set how the container connects to the network. DLNA requires `Host` mode to function.
```

> Any other settings can be left as default.

Reviewer 3: ```suggestion
Other settings can be left as default.
```
There should also be explainers on what other settings do for advanced users.

Your settings should look like this:

> ![Advanced Settings](/images/docs/install-synology-13.png)

Reviewer 3: Where is the directory for config files?

![Advanced Settings](/images/docs/install-synology-14.png)

> Click "Next".

Reviewer 3: ```suggestion
Click `Next` to proceed to the next step.
```

> You can review your settings and then click "Apply".
> Make sure the checkbox "Run this container after the wizard is finished" is checked.
> You can now see the container in the "Container" tab.

Reviewer 3: ```suggestion
Settings can be reviewed at this screen. Check the `Run this container after the wizard is finished` checkbox and click `Apply` if everything looks correct. The container should now be shown in the `Container` tab.
```

> Browse to `http://SERVER_IP:8096` to access the web client.
> If you used a different port, replace 8096 with the port you used.

Reviewer 3: ```suggestion
Browse to `http://SERVER_IP:8096` in a browser on a other device to finish setting up the Jellyfin server.
If a different port was used, replace `8096` with the port used instead.
```

> ## earlier versions of DSM

Reviewer 3: ```suggestion
## Earlier Versions of DSM
```

![Installing Synology](/images/docs/install-synology-9.png)

Browse to `http://SERVER_IP:8096` to access the web client.

> If you used a different port, replace 8096 with the port you used.

Reviewer 3: ```suggestion
If a different port was used, replace `8096` with the port used instead.
```
